dxr/plugins/python/indexers.py

```diff
@@ -233,13 +241,13 @@ def analyze_tokens(self):
         utf8_token_gen = tokenize.generate_tokens(
             StringIO(self.contents.encode('utf-8')).readline)
 
-        # These are a mapping from the utf-8 byte starting points provided by
-        # the ast nodes, to the unicode character offset tuples for both the
-        # start and the end points.
+        # This is a mapping from the utf-8 byte starting points provided by
```

Comment:
Also, since this table is what we return, it would be great to update the docstring, making it specific about what comes out of this function. Right now, it sounds like it does a lot more analysis than it does.